docs-src/Makefile

```diff
@@ -1,10 +1,24 @@
+L0_VERSION:=$(shell git describe --tags)
+export L0_VERSION_V:=""
```

Comment:
i dont think this is required

Reply:
I needed it for the expenv, otherwise the env wasn't being found. Not sure if it's a bug or something I did wrong.

Reply:
since, from our convo below, `expenv` commands will be using `L0_VERSION` instead of `L0_VERSION-V`, I think this can be removed?